backend/inspirehep/access_control.py

```diff
@@ -43,3 +43,13 @@ def can(self):
 
 def session_cataloger_permission_factory(*args, **kwargs):
```

Comment:
I would also change this while you're at it, by having a dummy `SessionCatalogerPermission.__init__`.

Reply:
@drjova Can I? :pray: :smile:

Reply:
🚀

Reply:
I agree `__init__` is better than a separate function.